### PR TITLE
Deaths, births and arrival ETL updates.  Add lookup sets

### PR DESCRIPTION
**Pull request by labkey-martyp**
#### Rationale
Deaths, births and arrival ETL updates.  Set up lookup sets and add country lookups.

#### Changes
- Incremental ETLs for Deaths and Births
- Update Arrival ETL to only pull from animal events
- Setup EHR lookup sets for TSV data
- Add country and country_category data to lookup sets

### CODE REVIEW

File: nirc_ehr/resources/data/country.tsv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+value	title	category
+1	St. Kitts	2
+2	Italy	2
+3	Japan	2
+4	"U.S.A. "	1
```
Comment (labkey-jeckels) [inline nirc_ehr/resources/data/country.tsv]: Why is there a trailing space here?

File: nirc_ehr/resources/views/populateData.html [hunk 2]
```diff
@@ -229,7 +250,44 @@
...
+                    success: this.onSuccess,
+                    failure: this.onError,
+                    scope: this
+                };
+
+                var origSuccess = config.success;
+                config.success = function (results, xhr, c) {
+                    console.log('lookup set records inserted');
+
+                    LABKEY.Ajax.request({
```
Comment (labkey-jeckels) [inline nirc_ehr/resources/views/populateData.html]: Is this required? I would have expected your changes here to make this unnecessary:

https://github.com/LabKey/ehrModules/pull/250

**Review by labkey-jeckels — APPROVED** [review]

File: nirc_ehr/resources/data/country.tsv (hunk 1 above)
Comment (labkey-martyp) [reply nirc_ehr/resources/data/country.tsv]: removed

File: nirc_ehr/resources/views/populateData.html (hunk 2 above)
Comment (labkey-martyp) [reply nirc_ehr/resources/views/populateData.html]: I forgot about that. This has been removed

**Pull request merged**